src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .flatMap(string -> Arrays.stream(string.split(",")))
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list"));
```

Comment:
```suggestion
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```
let's add input data to make message more informative